app/views/budgets/index.html.erb

```diff
@@ -0,0 +1,7 @@
+<h1><%= t(".title") %></h1>
+<ul>
+  <% @budgets.each do |budget| %>
+    <%= render "detail", budget: budget %>
+    </br>
+  <% end %>
```

Comment:
Same comments as in the other PR.
1. create a `app/views/budgets/_budget` partial
2. instead of lines 3-6, use `<%= render @budgets %>`

You should also check what happens when there are no budgets...

Reply:
Very magical! Done sha: a3fcca6